inflation/main.py

```diff
@@ -101,7 +143,19 @@ async def fetch_data(
     fetch_inflation_data(excel_path, OUTPUT_PATH, filename)
     return {
         "success": True,
-        "message": f"{BUCKET_NAME}/{filename} is preparing.",
+        "message": f"{CRAWLER_BUCKET}/{filename} is preparing.",
+    }
+
+
+@app.get("/Parse", tags=["Parse"])
+async def parse_data(
+    source_filename, destination_filename="inflation-resources/data/inflation/"
```

Comment:
`destination_filename` doesn't make sense. Let's not ask the user. It is not his/her business.

Reply:
pasam gormuyor musun bunu mesela